kafka/src/main/java/com/bazaarvoice/emodb/kafka/JsonPOJOSerde.java

```diff
@@ -0,0 +1,98 @@
+package com.bazaarvoice.emodb.kafka;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.kafka.common.errors.SerializationException;
+import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serializer;
+
+import java.util.Map;
+
+public class JsonPOJOSerde<T> implements Serde<T> {
+
+    private final ObjectMapper mapper = new ObjectMapper();
```

Comment:
should we set this field from constructor?

Reply:
I can definitely set it from the constructor. I just don't want to make the ObjectMapper required to provide to the Serde. The existing Emo ObjectMapper is definied in `CustomJsonObjectMapperFactory`, and has lots of extra modules that I don't want the megabus to build dependence on.

Reply:
I am moving assignment into the constructor